src/ExadelMentorship.ConsoleApp/Program.cs

```diff
@@ -1,33 +1,14 @@
 ﻿// See https://aka.ms/new-console-template for more information
-using ExadelMentorship.BusinessLogic.Exceptions;
+
+using ExadelMentorship.BusinessLogic;
 using ExadelMentorship.BusinessLogic.Features.WeatherFeature;
-using ExadelMentorship.BusinessLogic.Models;
 using ExadelMentorship.IntegrationTests;
-using Microsoft.Extensions.DependencyInjection;
-using System;
+
 
 var weather = DI.Resolve<Weather>();
```

Comment:
1. Weather would be MainJob's dependency via constructor, so no need for manual intervention.
2. Be mindful calling `DI.Resolve` as it reconstructs the container for each call.